rel/overlay/etc/default.ini

```diff
@@ -69,8 +69,9 @@ default_engine = couch
 ; is 10GiB. A value of 0 or less will disable partition size checks.
 ;max_partition_size = 10737418240
 ;
-; Start node in single_node mode so default databases are created immediately.
-;single_node = true
+; When true, system databases _users and _replicator are created immediately
```

Comment:
minor nit - `_global_changes` is also counted as a system database (perhaps questionably) so will get created automatically.

Reply:
@janl do you want to remove `_global_changes` from the setup wizard for 3.0.0?

let me know and we can decide if we fix the message here, or `src/setup/erl/setup.src`. I can see an argument both ways.

Reply:
Have moved this change out to another PR.